scripts/local-generate-svg.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export SOURCECRED_DIRECTORY=$(pwd)/sourcecred_data
+SOURCECRED_GITHUB_TOKEN=`cat secrets/token`
+REPOS=`cat repositories.txt`
```

Comment:
We should probably check here that this secrets file exists.

Reply:
And then of course exit if the variables aren't what we expect.

Reply:
The `local-*` scripts are mostly debugging helpers. But you're right it would be useful for others if it's consistent and produces clear errors like this. Going to improve those a little.